src/main/java/reposense/authorship/FileInfoExtractor.java

```diff
@@ -69,13 +71,24 @@
         if (!lastCommitHash.isEmpty()) {
             fileInfos = getEditedFileInfos(config, lastCommitHash);
         } else {
-            getAllFileInfo(config, fileInfos);
+            fileInfos = getAllFileInfo(config, false);
         }
 
         fileInfos.sort(Comparator.comparing(FileInfo::getPath));
         return fileInfos;
     }
 
+    /**
+     * Extracts a list of relevant binary files given in {@code config}.
+     */
+    public static List<FileInfo> extractBinaryFileInfos(RepoConfiguration config) {
+        logger.info(String.format(MESSAGE_START_EXTRACTING_BINARY_FILE_INFO, config.getLocation(), config.getBranch()));
+
+        List<FileInfo> binaryFileInfos = getAllFileInfo(config, true);
+        binaryFileInfos.sort(Comparator.comparing(FileInfo::getPath));
+        return binaryFileInfos;
+    }
```

Comment:
If the user only wants to analyze between any two periods of time, and that binary files were not modified between the two specified times (note that these binary files could exist before the analysis start date), will this method capture that as well?

If so, shouldn't we exclude such binary files since it wasn't modified before?

Reply:
The current method would capture all binary files, regardless of whether it was modified.

However, the actual date range filtering is done under `GitLog#getBinaryFileAuthors`, which would return an empty string if it was not modified within the specified time.